### PR TITLE
Style cookies banner

### PR DESCRIPTION
**Pull request by ErinHales**
*[GitHub keywords to close any associated issues](https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)*

## Motivation

Low contrast ratio on the old banner.

## Changelog

 - Better contrast ratio
 - Updated text
 - Updated styling
 - Matches celo.org cookie banner

### Incompatible Changes

None
Screen Shot 2021-03-01 at 10 15 15 AM
Screen Shot 2021-03-01 at 10 15 30 AM

### CODE REVIEW

File: apps/block_scout_web/lib/block_scout_web/templates/layout/_cookie_banner.html.eex [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="cookie-banner" hidden data-selector="cookie-banner">
+  <div class="cookie-banner-content p-4">
+    <div>
+    <%= gettext("Allow tracking of my activity on ") %>
```
Comment (aaronmgdr) [inline apps/block_scout_web/lib/block_scout_web/templates/layout/_cookie_banner.html.eex:4]: nitpick but indent file lines?

**Review by aaronmgdr — APPROVED** [review]
good one

**Review by aaronmgdr — COMMENTED** [review]
is the package-lock.json right? seems like a lot of lines

**Comment by ErinHales** [comment]
> is the package-lock.json right? seems like a lot of lines

Hm, that is surprising.  All I did was `npm i`... I deleted and reinstalled and I'm still seeing a lot of lines added.

**Comment by aaronmgdr** [comment]
> > is the package-lock.json right? seems like a lot of lines
> 
> Hm, that is surprising. All I did was `npm i`... I deleted and reinstalled and I'm still seeing a lot of lines added.

🤷🏻‍♂️ maybe before npm install was ran a directory up? i dont really know how this works but seems like there the web is a subpackage?

**Review by yourbuddyconner — APPROVED** [review]
Didn't render it myself but screenshots look good and code content LGTM.

**Pull request merged**